.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -30,10 +30,19 @@ body:
 
         ```
 
+  - type: textarea
+    id: traceback
+    attributes:
+      label: Traceback
+      description: |
+          Error messages and stack traces.
+          Please use the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.
+      value: |
         ```bash
         Traceback (most recent call last): File ...
 
         ```
```

Comment:
if you want the rendering of https://github.com/DLR-RM/stable-baselines3/pull/1111#issuecomment-1277865049, use 

```
  - type: textarea
    id: traceback
    attributes:
      label: Traceback
      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
      placeholder: "Traceback (most recent call last): File ..."
      render: shell